pootle/apps/import_export/views.py

```diff
@@ -64,10 +64,12 @@ def export(request):
         return download(f.getvalue(), "%s.zip" % (prefix), "application/zip")
 
 
-def handle_upload_form(request):
+def handle_upload_form(project, request):
```

Comment:
I think it's preferable (although not necessary) to add any new parameters at the end. I'd still consider doing that to keep `request` as the first parameter, which is a kinda convention for Django code dealing with views.